braglog/cli.py

```diff
@@ -157,13 +157,29 @@ def show(  # noqa: PLR0913,PLR0912
 
     order = LogEntry.log_date.desc() if reverse else LogEntry.log_date.asc()
     entries = entries.order_by(order) if entries else entries
+
     if edit:
         for entry in entries:
-            message = click.edit(entry.message)
-            if message is not None:
-                entry.message = message.rstrip("\n")
+            text = f"{entry.log_date.strftime('%Y-%m-%d')}: {entry.message}"
```

Comment:
With this PR, now we can edit the date of a message previously we couldn't :exploding_head: